plone/app/z3cform/templates/date_time_input.pt

```diff
@@ -0,0 +1,12 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      tal:omit-tag=""
+>
+  <input name="${view/name}"
+         value="${view/value}"
+         tal:attributes="
+           view/attributes;
+         "
+  />
+  <!--! NOTE: we generate the input type in widget attributes -->
```

Comment:
```suggestion
  <tal:c condition="nothing">NOTE: we generate the input type in widget attributes</tal:c>
```

Reply:
This is not needed as the `<!--! --> ` (note the second exclamation mark) is a chameleon chortcut for dropping comments on output ... see https://chameleon.readthedocs.io/en/latest/reference.html#markup-comments